src/authorize.js

```diff
@@ -37,6 +37,27 @@
       url += '&state=' + encodeURIComponent(redirectUri.substring(hashPos+1));
     }
     url += '&response_type=token';
+
+    if (global.cordova) {
+      return RemoteStorage.Authorize.openWindow(
+          url,
+          redirectUri,
+          'location=no,clearsessioncache=yes,clearcache=yes'
+        )
+        .then(function (authResult) {
+          remoteStorage.remote.configure({
+            token: authResult['access_token']
```

Comment:
Ok I'm not sure what I should do about this now...

I had authResult.access_token before and it complained that it's not in camelCase. =/